src/libstd/net/ip.rs

```diff
@@ -769,7 +769,16 @@ impl FromInner<c::in_addr> for Ipv4Addr {
 
 #[stable(feature = "ip_u32", since = "1.1.0")]
 impl From<Ipv4Addr> for u32 {
-    /// It performs the conversion in network order (big-endian).
+    /// Convert an `Ipv4Addr` into a host byte order `u32`.
```

Comment:
admittedly i'm not sure what this terminology should be here w.r.t. the removal of 'network order' and addition of 'host byte order'. i just took it as a suggestion from [this thread](https://github.com/rust-lang/rust/issues/48819). might be nice to get input from the libs/docs team here @rust-lang/libs @rust-lang/docs

Reply:
To me the new wording seems good, and more easily understandable than the old version, although I'd be curious to hear what other people think.

Reply:
I agree. The new wording looks good, and this is how I'd expect this function to behave.